tests/__init__.py

```diff
@@ -1,25 +1,15 @@
 #! /usr/bin/env python
 # vim: expandtab shiftwidth=4 softtabstop=4 tabstop=17 filetype=python :
 
-# Make things as three-ish as possible (requires python >= 2.6)
-from __future__ import (unicode_literals, print_function,
-                        absolute_import, division)
-# Namespace cleanup
-del unicode_literals, print_function, absolute_import, division
-
-#
-# ----- End header -----
-#
-
 import unittest
 import transaction
 
 from caramel.models import (
     init_session,
     DBSession,
-    )
+)
 
-from . import fixtures
+from tests import fixtures
```

Comment:
This is a functional change that may not be intended.   it assumes "tests" is an installed / available python package rather than being a local import.

Reply:
I cannot for the life of me remember exactly what it was, but a linter or debugger got caught up on it, and since the "from 'local package' import module" semantic is used in multiple cases of the code thats what i substituted it for.